code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -359,30 +380,38 @@ GLOBAL_LIST_EMPTY(scp173s)
 		if(do_after(user, 10 SECONDS, dropping) && loc == oloc)
 			dropping.forceMove(src)
 			underlays += image(dropping)
-			visible_message(SPAN_NOTICE("[user] puts SCP-173 in the cage."))
-			name = "SCP-173 Cage"
+			desc = "A cage for containing SCP-173. It is currently holding [dropping]."
+			visible_message(SPAN_NOTICE("[user] puts [dropping] in the cage."))
 			playsound(loc, 'sound/machines/bolts_down.ogg', 50, 1)
+			icon_state = "1"
 			return TRUE
 		return FALSE
 	if(isliving(dropping))
 		to_chat(user, SPAN_WARNING("\The [dropping] won't fit in the cage."))
 	return FALSE
 
-/obj/structure/scp173_cage/attack_hand(mob/living/carbon/human/H)
+/obj/structure/scp173_cage/attack_hand(mob/living/A)
 	if(!LAZYLEN(contents))
 		return ..()
-	visible_message(SPAN_WARNING("[H] attempts to open \the [src]."))
-	if(do_after(H, 5 SECONDS, src))
-		visible_message(SPAN_DANGER("[H] opens \the [src]!"))
+	if(istype(A, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = A
+		visible_message(SPAN_WARNING("[H] attempts to open \the [src]."))
+		if(do_after(H, 5 SECONDS, src))
+			visible_message(SPAN_DANGER("[H] opens \the [src]!"))
+			ReleaseContents()
+	else if(istype(A, /mob/living/scp_173))
+		var/mob/living/scp_173/S = A
 		ReleaseContents()
```

Comment:
hahahaha 173s can release other 173s 
tho why is `var/mob/living/scp_173/S = A` needed?
also I'd add a check to make sure the 173 that is releasing is *not* being watched / not caged itself

Reply:
that isn't done its just for testing purposes, currently working on it